comment/comment.go

```diff
@@ -0,0 +1,45 @@
+package comment
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	comment_model "dat.havinh/week5-assignment/GrabGoTrainingWeek5Assignment/comment/model"
+	httpclient "dat.havinh/week5-assignment/GrabGoTrainingWeek5Assignment/http-client"
+)
+
+const (
+	getCommentsEndpoint = "https://my-json-server.typicode.com/typicode/demo/comments"
+)
+
+type CommentGetter interface {
+	GetComments() ([]comment_model.Comment, error)
+}
+
+type CommentGetterImpl struct {
+	httpClient httpclient.HTTPClient
+}
+
+func (commentGetter *CommentGetterImpl) GetComments() ([]comment_model.Comment, error) {
```

Comment:
Nitpick: this function does 2 things (get data and rendering), so can be separated into 2 smaller functions for readability